tests/Nodes/NodeTypeTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace SVG\Nodes;
+
+use Illuminate\Support\Str;
+use PHPUnit\Framework\TestCase;
+use SVG\Nodes\Shapes\SVGRect;
+use ValueError;
+
+class NodeTypeTest extends TestCase
+{
+    /** @dataProvider provideCases */
+    public function testCases($nodeTypeFQN)
+    {
+        $SVGNodeFqn = SVGNode::class;
+        $this->assertTrue(is_a($nodeTypeFQN, $SVGNodeFqn, true), "$nodeTypeFQN is not a $SVGNodeFqn");
```

Comment:
I think this is useful, as far as I can tell every "node type" should be a SVGNode. This test will further reduce chance for typos.